### PR TITLE
Improve some tooling around version reporting

### PR DESCRIPTION
**Pull request by bryevdv**

### CODE REVIEW

**Comment by bryevdv** [comment]
renamed env vars:

`BOKEH_LOCAL_DOCS_CDN` -> `BOKEH_DOCS_CDN`

`BOKEH_RELEASED_DOCS` -> `BOKEH_DOCS_VERSION`

File: bokeh/settings.py [hunk 1]
```diff
@@ -54,11 +54,11 @@ def rootdir(self, default=None):
...
 
-    def local_docs_cdn(self, default=None):
-        return self._get_str("LOCAL_DOCS_CDN", default)
+    def docs_cdn(self, default=None):
+        return self._get_str("DOCS_CDN", default)
 
-    def released_docs(self, default=None):
-        return self._get_bool("RELEASED_DOCS", default, False)
+    def docs_version(self, default=None):
+        return self._get_str("DOCS_VERSION", default, None)
```
Comment (damianavila) [inline bokeh/settings.py]: You don't need to pass `None` here...

File: sphinx/source/conf.py [hunk 2]
```diff
@@ -61,20 +61,20 @@
...
 # and |release|
-from bokeh import __version__
-
-# Check for version override (e.g. when re-deploying a previously released
-# docs, or when pushing test docs that do not have a corresponding BokehJS
-# available on CDN)
-from bokeh.settings import settings
-if settings.released_docs():
-    __version__ = __version__.split('-')[0]
+from bokeh import __version__, __base_version__
```
Comment (damianavila) [inline sphinx/source/conf.py]: why you are importing here `__base_version__`, it is not used anywhere...

**Comment by damianavila** [comment]
You need to update here the envs: http://bokeh.pydata.org/en/latest/docs/dev_guide/notes.html as well...

File: sphinx/source/conf.py (hunk 2 above)
Comment (bryevdv) [inline sphinx/source/conf.py]: leftover cruft from earlier experiment thanks

**Comment by bryevdv** [comment]
@damianavila I do want to update there, but... I also want to redploy 0.9 docs probably a few times... realistically, long term we probably need to keep release branches around so doc updates can happen independently of new dev on master, and then get pulled into master. Let's see about getting more server info into 0.9 docs before adding new/incompat info.

**Comment by bryevdv** [comment]
@damianavila any other comments?

**Comment by bryevdv** [comment]
@damianavila I am planning on merging this later today unless there are other comments

**Comment by damianavila** [comment]
LGTM

**Pull request merged**